Dockerfile

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Stage 1: Build the cp-utility binary
+FROM rust:1.68 as builder
+
+# possible values: x86_64 | aarch64
+ARG ARCH=x86_64
```

Comment:
Where does this get overriden in the build process? Should we be building on both x86/amd64?

Reply:
Does buildx automatically pass this in for different archs?

Reply:
it will build, but we need to pass the `ARCH` argument explicitly. I will modify the workflow.